packages/promotion/src/models/application-method.ts

```diff
@@ -35,25 +35,25 @@ export default class ApplicationMethod {
   id!: string
 
   @Property({ columnType: "numeric", nullable: true, serializer: Number })
-  value?: number | null
+  value?: string | null
 
   @Property({ columnType: "numeric", nullable: true, serializer: Number })
-  max_quantity?: number | null
+  max_quantity?: string | null
```

Comment:
comment: Depends on discussion in other PR

Reply:
switched backed to number